EventFilter/L1TRawToDigi/interface/Unpacker.h

```diff
@@ -12,6 +12,13 @@ namespace l1t {
       public:
          virtual bool unpack(const Block& block, UnpackerCollections *coll) = 0;
```

Comment:
Can you add a constructor that sets algoVersion_(0) please?

Reply:
Can probably be done in the .h file since it's only one line.

Reply:
@thomreis Updated.